src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/PropertyPage.cs

```diff
@@ -82,15 +82,15 @@ protected bool IsDirty
         /// </summary>
         private T WaitForAsync<T>(Func<Task<T>> asyncFunc)
         {
-            return _threadHandling.ExecuteSynchronously<T>(asyncFunc);
+            return _threadHandling != null ? _threadHandling.ExecuteSynchronously<T>(asyncFunc) : default(T);
```

Comment:
Hmm that doesn't seem right, you no longer execute the func. There's already a bug on this - I'd rather a fix that fixes calling this method in those cases.